great_tables/_utils_nanoplots.py

```diff
@@ -913,7 +913,9 @@ def _generate_nanoplot(
     # there are no x values, generate equally-spaced x values according
     # to the number of y values
     if plot_type == "line" and x_vals is not None:
-        if expand_x is not None and _val_is_str(expand_x):
+        if isinstance(expand_x, str) or (
```

Comment:
I don't understand this code. The type annotation for expand_x in this function and fmt_nanoplot does not include strings. Which of these functions allows strings? (and which should / shouldn't?)

Reply:
Shoot, I'm realizing that this is just a guard, which raises a error by design. I think the big issue is that this PR does not cause the failing example to succeed on the guide page

Reply:
Ah, the issue is that `_val_is_str` raises an error on lists!